src/curve/stable/StablePair.sol

```diff
@@ -43,12 +43,14 @@ contract StablePair is ReservoirPair {
 
     constructor(address aToken0, address aToken1) Pair(aToken0, aToken1)
     {
-        ampData.initialA        = factory.read("ConstantProductPair::amplificationCoefficient").toUint64() * uint64(StableMath.A_PRECISION);
+        ampData.initialA        = factory.read("SP::amplificationCoefficient").toUint64() * uint64(StableMath.A_PRECISION);
         ampData.futureA         = ampData.initialA;
         // perf: check if intermediate variable is cheaper than two casts (optimizer might already catch it)
         ampData.initialATime    = uint64(block.timestamp);
         ampData.futureATime     = uint64(block.timestamp);
 
+        swapFee = uint256(factory.get(keccak256("SP::swapFee")));
+
```

Comment:
Can be DRY

Reply:
fixed, also changed all `.get` to the more civilized `.read()`